app/views/providers/application_merits_task/has_other_involved_children/show.html.erb

```diff
@@ -20,7 +20,9 @@
             head.with_row do |row|
               row.with_cell(text: "Name")
               row.with_cell(text: "Date of birth")
-              row.with_cell(colspan: 2, scope: "colgroup", text: "Action")
+              row.with_cell(colspan: 2) do
+                tag.span("Change or remove involved child", class: "govuk-visually-hidden")
+              end
```

Comment:
semantic: could we re-apply the `scope` attribute to the cell?

nit: could we add the option  text and `html_attributes: { class: "govuk-visually-hidden" }` options to the cell, then we could remove the additional `span` tag